thirdeye-ui/src/app/pages/alerts-create-page/alerts-create-copy-page.component.tsx

```diff
@@ -29,11 +29,20 @@ export const AlertsCreateCopyPage: FunctionComponent = () => {
     useEffect(() => {
         getAlert(Number(alertId)).then((alert) => {
             if (alert) {
+                const keysToDelete: (keyof Alert)[] = [
+                    "id",
+                    "lastTimestamp",
+                    "active",
+                    "owner",
+                    "created",
+                    "updated",
+                ];
```

Comment:
Probably best to make this a constant before the declaration of the component at line 22 so we don't have to reinitialize this every time

Reply:
I moved the whole copying logic (the deletion code and these keys) to an accompanying utils file to hide the implementation detail, and to allow for future reuse if needed.